pkg/build/controller/strategy/util.go

```diff
@@ -463,10 +463,26 @@ func setupBuildCAs(build *buildv1.Build, pod *corev1.Pod, includeAdditionalCA bo
 				Name: buildapihelpers.GetBuildCAConfigMapName(build),
 			},
 		}
-		if includeAdditionalCA {
+		// Mount the service signing CA key for the internal registry.
+		// This will be injected into the referenced ConfigMap via the openshift/service-ca-operator, and block
+		// creation of the build pod until it exists.
+		//
+		// See https://github.com/openshift/service-ca-operator
```

Comment:
You should reference openshift/service-serving-cert-signer instead. We backed out of the renaming of this component, so sorry for the confusion with these two repos..